scripts/upload.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/sh
+cd ..
+# test
+printf "\033[1;35mtesting...\033[0m\n"
+if python setup.py test
+then
+    printf "\033[1;35mtesting successful\033[0m\n"
+else
+    printf "\033[1;35mtests failed---aborting\033[0m\n"
+    exit 1
+fi
+# build
+printf "\033[1;35mbuilding...\033[0m\n"
+python setup.py sdist
+python setup.py bdist_wheel --universal
+# upload
+printf "\033[1;35muploading...\033[0m\n"
+while true; do
+        read -p "upload to PyPI? [y/n]: " yn
```

Comment:
I think in this case I like having a required answer here, because uploading is really something I want people to be conscious of.

If we went with a default I would make the default N.